script.js

```diff
@@ -49,18 +41,17 @@ let foundPairs = 0;
 /**
  * Dynamically create the divs for words and kanjis and append them to the HTML.
  * @param  wordPairs - an array of pairs of words and kanjis, like [['word1', 'translation1'] ...
- * @param  nPairs - the number of pairs to create
+ * @param  pairRenderLimitIndex - index of the last pair to render + 1, incremented by pairCount in checkIfWon()
```

Comment:
The docstring of one function should not describe the insides of another function ( "`, incremented by pairCount in checkIfWon()`"). This is a code smell known as "looking into someone else's underwear" . We try to avoid that here :D Functions should be as independent as possible.

I understand this is tempting to do from the learners' point of view, but in a bigger project this could easily get out of hand. E. g. if you made changes in `checkIfWon` you would have to remember to update this docstring too. It can also be a sign of spaghetti logic.